### PR TITLE
updated to 1.19

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

Wooohooo 1.19
This pull request updates slimefun to 1.19

## Proposed changes

Added the commons-lang depedency since this is removed from 1.19
We do want to change this out in a fast follow to our own validation.
This pull request is blocked by  https://github.com/baked-libs/dough/pull/176

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [x] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
There are some illegal values in the biomes file, like #is_end

**Comment by ybw0014** [comment]
WorldEdit's repo has been changed to: https://maven.enginehub.org/repo

**Comment by J3fftw1** [comment]
> WorldEdit's repo has been changed to: https://maven.enginehub.org/repo

This has been updated on dough side already. For some reason mypet is just failing now

**Comment by TheBusyBiscuit** [comment]
>This has been updated on dough side already.

Neither were updated to the new repository urls.
I went ahead and did it for both, dough and Slimefun now.
Both are compiling just fine again.

https://github.com/Slimefun/Slimefun4/commit/8ecda680e9c2719c5ed0f5eaa7ede8c4a2e3738b
https://github.com/baked-libs/dough/commit/28b81361a1ca26b643db9956979b5f1b5cfe0da0

**Comment by J3fftw1** [comment]
not sure why @TheBusyBot says there are still merge conflicts.

File: pom.xml [hunk 1]
```diff
@@ -502,5 +502,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Todo remove this dependency -->
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>2.6</version>
+        </dependency>
```
Comment (TheBusyBiscuit) [inline pom.xml:510]: What do we need this for though?

Comment (WalshyDev) [reply pom.xml:510]: We use it everywhere in Slimefun currently. All the Validate's are from commons lang, however Spigot now removed it so here we are.

To keep this change small and quick, I told Jeff to just add this for now and we can refactor as a fast follow.

Comment (TheBusyBiscuit) [reply pom.xml:510]: Ah, `Validate`, I see

**Review by WalshyDev — APPROVED** [review]

**Comment by WalshyDev** [comment]
Blocked on dough being merged and Pom updated

**Review by TheBusyBiscuit — APPROVED** [review]
👍🏻 LGTM.

I deployed dough 1.2 to `maven-central` and will update the pom in an internal branch.
Thanks a lot, Panda!

**Pull request merged**